packages/presentation/lib/screen/details/details_bloc.dart

```diff
@@ -27,6 +36,13 @@ class _DetailsBloc extends BlocImpl<DetailsScreenArguments, DetailsData>
     _updateData();
   }
 
+  @override
+  Future<void> shareMovie(BuildContext context) async {
```

Comment:
Seems that BuildContext dependency is not good place for Bloc, @sergei-mikhailovskii-idf  make it clear please.

Reply:
agree

Reply:
fixed